src/web-api/matchMedia.ts

```diff
@@ -1,8 +1,14 @@
 import { fromEventPattern, Observable } from 'rxjs';
 
+/**
+ * A RxJS operator for getting the results of the
+ * {@link https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList|Media Query API}
+ * @param mediaQueryList The media query list to observe
+ * @returns An Observable of events from the media query
+ */
 function fromMediaListQuery(mediaQueryList: MediaQueryList): Observable<MediaQueryListEvent> {
   return fromEventPattern(
-    handler => mediaQueryList.addListener(handler),
-    handler => mediaQueryList.removeListener(handler)
+    handler => mediaQueryList.addEventListener('change', handler),
```

Comment:
how do you know about this 'change' parameter, I didn't found anything about that in the documentation?

Reply:
in `lib.dom.d.ts` is the following:

![image](https://user-images.githubusercontent.com/376930/76114388-3f757300-5fe6-11ea-9eb3-cd154a63fab7.png)